datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/catalog.py

```diff
@@ -0,0 +1,101 @@
+import csv
+import os
+import tempfile
+
+import click
+
+from ...utils import (
+    get_assets_directory,
+    get_check_file,
+    get_config_file,
+    get_data_directory,
+    get_valid_integrations,
+    has_e2e,
+)
+from ..console import CONTEXT_SETTINGS, abort, echo_info
+
+CSV_COLUMNS = ['name', 'has_dashboard', 'has_logs', 'is_jmx', 'is_prometheus', 'is_http', 'has_e2e', 'tile_only']
+DOGWEB_DASHBOARDS = ('vsphere', 'sqlserver', 'tomcat', 'pusher', 'sigsci', 'marathon', 'ibm_was', 'nginx', 'immunio')
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Create a catalog with information about integrations')
+@click.argument('checks', nargs=-1, required=True)
```

Comment:
With #5674 merged, you can add autocompletion argument to this now!

```suggestion
@click.argument('checks', nargs=-1, autocompletion=complete_valid_checks,  required=True)
```